app/models/badges/functional_february_badge.rb

```diff
@@ -0,0 +1,25 @@
+module Badges
+  class FunctionalFebruaryBadge < Badge
+    TRACK_SLUGS = %w[
+      elixir erlang fsharp haskell ocaml scala sml gleam
+    ].freeze
+
+    seed "Functional February",
+      :ultimate,
+      'functional',
+      'Completed five exercises in a functional language in February'
+
+    def self.worth_queuing?(exercise:)
+      TRACK_SLUGS.include?(exercise.track.slug)
+    end
+
+    def award_to?(user)
+      user.solutions.completed.joins(exercise: :track).
+        where('tracks.slug': TRACK_SLUGS).
+        where('MONTH(completed_at) = 2').
+        count >= 5
+    end
```

Comment:
Just as a check: we don't only award this badge to people that are participating in the challenge?

Reply:
Correct. But thanks for checking!